chaste_codegen/_chaste_printer.py

```diff
@@ -6,7 +6,7 @@
     S,
 )
 from sympy.core.mul import _keep_coeff
-from sympy.printing.cxxcode import cxxcode
+from sympy.printing import cxxcode
```

Comment:
What's the minimum version this works with? Is that version supplied with oldest LTS ubuntu supported by chaste?

Reply:
we don't support < 1.6.1 (there were some other things that didn't work properly with older versions). It's all python nothing to do with ubuntu packages.